README.md

```diff
@@ -176,6 +176,11 @@ $ limactl start --name=default template://docker
 > NOTE: `limactl start template://TEMPLATE` requires Lima v0.9.0 or later.
 > Older releases require `limactl start /usr/local/share/doc/lima/examples/TEMPLATE.yaml` instead.
 
+To create an instance "default" with modified parameters:
+```console
+$ limactl start --set='.cpus = 2 | .memory = "2GiB"'
+```
+
```

Comment:
I wonder if we need to include a reference to the `yq` documentation, so people can learn to construct more powerful expressions for e.g. scripting:

```console
$ limactl start --set '.env.HOSTPATH=strenv(PATH)' template://alpine
```

Reply:
I expected the same syntax to be available for the `limactl edit` command as well.

Reply:
I guess I should have specified: I would expect `limactl edit --set ...` to just edit the config, but not to invoke an editor.

It would just be a mechanism to modify the config programmatically without starting the instance.

Reply:
We can use something like `--eval`, since that is the name of the `yq` command being invoked ?

```
Available Commands:
  completion       Generate the autocompletion script for the specified shell
  eval             (default) Apply the expression to each document in each yaml file in sequence
  eval-all         Loads _all_ yaml documents of _all_ yaml files and runs expression once
  help             Help about any command
  shell-completion Generate completion script
```

I agree that `--set` is somewhat misleading, and that `--yq` might be a bit cryptic for lima users...

Adding it to `edit` should be straight-forward.

Reply:
>  I would expect limactl edit --set ... to just edit the config, but not to invoke an editor.

:+1:

`--eval` sounds much more misleading

Reply:
`limactl edit --set` can be another PR if complicated

Reply:
The error handling can be a bit tricky to do, so I'd like that.